action/build/cancel.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package build
+
+import (
+	"io/ioutil"
+
+	"github.com/go-vela/cli/internal/output"
+
+	"github.com/go-vela/sdk-go/vela"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Cancel cancels a build based off the provided configuration.
+func (c *Config) Cancel(client *vela.Client) error {
+	logrus.Debug("executing cancel for build configuration")
+
+	logrus.Tracef("canceling build %s/%s/%d", c.Org, c.Repo, c.Number)
+
+	// send API call to cancel a build
+	//
+	// https://pkg.go.dev/github.com/go-vela/sdk-go/vela?tab=doc#BuildService.Cancel
+	_, resp, err := client.Build.Cancel(c.Org, c.Repo, c.Number)
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	// Read Response Body
+	respBody, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+
+	respS := string(respBody)
```

Comment:
I think we might want to change this a bit.

Typically we don't attempt to read the response body unless needed for debugging/troubleshooting purposes.

It would be better to simplify this to:

```suggestion
	// send API call to cancel a build
	//
	// https://pkg.go.dev/github.com/go-vela/sdk-go/vela?tab=doc#BuildService.Cancel
	build, _, err := client.Build.Cancel(c.Org, c.Repo, c.Number)
	if err != nil {
		return err
	}
```

Reply:
agree and updated.